models/cocktail-model.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-var mongoose = require('mongoose');
+const mongoose = require('mongoose');
```

Comment:
You've probably already seen this: `const` for variables that can never be reassigned; `let` for variables that can be. Both are block scoped (to the surrounding curly braces), not scoped to the surrounding function. `var` is now a special case, to be used only when we _need_ function scoping.

P.S. One gotcha with `const`: it looks immutable, but it's not. You can't assign it a _new_ value, but you can modify an existing one:
```
const myVar = {
  key: 'value'
};
myVar.key = 'someNewValue'; // Looks illegal, but isn't, because we haven't reassigned `myVar` itself
```